### PR TITLE
Health Database Optimizations

### PR DESCRIPTION
**Pull request by yakloinsteak**
## Description

https://github.com/greenriver/devops-requests/issues/739

I tested these changes against the problematic health database, but somebody with more knowledge of the application and business logic should also test this before merging. 

## Type of change
- [x] Query speed-ups

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [ ] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

**Review by awisema — APPROVED** [review]
Thanks Todd! These look reasonable to me.

**Comment by yakloinsteak** [comment]
@eanders Can you take a look and merge it if you're okay with it as well?

File: app/models/health/release_form.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@ class ReleaseForm < HealthBase
     scope :recent, -> { order(signature_on: :desc).limit(1) }
     scope :reviewed, -> { where.not(reviewed_by_id: nil) }
     scope :valid, -> do
-      parent_ids = Health::ReleaseFormFile.where.not(parent_id: nil).pluck(:parent_id)
+      parent_ids = Health::ReleaseFormFile.where.not(parent_id: nil).select(:parent_id)
```
Comment (eanders) [inline app/models/health/release_form.rb:40]: Pretty sure this is safe, but @awisema can you confirm these parent IDs are going to be used in the same database?  With so many cross-database situations, just want to double check a `select` is ok here.

**Review by eanders — APPROVED** [review]
Looks good if we confirm the select is in the same DB.

**Comment by yakloinsteak** [comment]
I ran this code in production and it worked.

File: app/models/health/release_form.rb (hunk 1 above)
Comment (awisema) [reply app/models/health/release_form.rb:40]: I have confirmed that the parent of a `Health::ReleaseFormFile` is the `Health::ReleaseForm` it is uploaded against -- so, there should be no x-db issue here.

Comment (eanders) [reply app/models/health/release_form.rb:40]: Thanks

**Pull request merged**